JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/ViewModels/RegisterViewModel.cs

```diff
@@ -0,0 +1,212 @@
+        public IMvxCommand ChangeStateCommand { get; }
+
+        public bool Active
+        {
+            get => _active;
+            set
+            {
+                _active = value;
+                RaisePropertyChanged(() => Active);
+            }
+        }
+
+        public string Username
+        {
+            get => _username;
+            set
+            {
+                _username = value;
+                RaisePropertyChanged(() => Username);
+                ReloadActive();
+            }
+        }
+
+        public string Password
+        {
+            get => _password;
+            set
+            {
+                _password = value;
+                RaisePropertyChanged(() => Password);
+                ReloadActive();
+            }
+        }
+
+        public string Email
+        {
+            get => _email;
+            set
+            {
+                _email = value;
+                RaisePropertyChanged(() => Password);
+                ReloadActive();
+            }
+        }
+
+        public bool RememberMe
+        {
+            get => _rememberMe;
+            set
+            {
+                _rememberMe = value;
+                RaisePropertyChanged(() => RememberMe);
+            }
+        }
+
+        public bool Error
+        {
+            get => _error;
+            set
+            {
+                _error = value;
+                RaisePropertyChanged(() => Error);
+            }
+        }
+
+        public string ConfirmPassword
+        {
+            get => _confirmPassword;
+            set
+            {
+                _confirmPassword = value;
+                RaisePropertyChanged(() => ConfirmPassword);
+                ReloadActive();
+            }
+        }
+
+        public bool ErrorMail
+        {
+            get => _errorMail;
+            set
+            {
+                _errorMail = value;
+                RaisePropertyChanged(() => ErrorMail);
+            }
+        }
+
+        public bool ErrorLogin
+        {
+            get => _errorLogin;
+            set
+            {
+                _errorLogin = value;
+                RaisePropertyChanged(() => ErrorLogin);
+            }
+        }
+
+        public bool Success
+        {
+            get => _success;
+            set
+            {
+                _success = value;
+                RaisePropertyChanged(() => Success);
+                RaisePropertyChanged(() => NotSuccess);
+            }
+        }
+
+        public bool NotSuccess
+        {
+            get => !_success;
+        }
+
+        public RegisterViewModel(IMvxNavigationService navigationService, IAuthenticationService authenticationService, IRegisterService registerService)
+        {
+            _navigationService = navigationService;
+            _authenticationService = authenticationService;
+            _registerService = registerService;
+
+            Error = false;
+
+            SignUp = new MvxCommand(async () =>
```

Comment:
use method

Reply:
Moved everything into a method in commit 6e4406d